hs_core/views/resource_ticket_rest_api.py

```diff
@@ -0,0 +1,194 @@
+import os
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.exceptions import PermissionDenied, NotFound
+from rest_framework import status
+
+from django.core.exceptions import SuspiciousFileOperation, ValidationError
+
+from hs_core.views import utils as view_utils
+from hs_core.views.utils import ACTION_TO_AUTHORIZE
+
+from django_irods.icommands import SessionException
+
+# hsapi/resource/{pk}/ticket/read/{path}
+#   GET: create
+# hsapi/resource/{pk}/ticket/write/{path} (currently disabled)
+#   GET: create
+# hsapi/resource/{pk}/ticket/bag
+#   GET: create
+# hsapi/resource/{pk}/ticket/info/{tnumber}
+#   GET: list
+#   DELETE: remove
+
+
+class CreateResourceTicket(APIView):
+    """
+    Create resource iRODS tickets via REST
+
+    REST URL: hsapi/resource/{pk}/{op}/{path}/
+    HTTP methods: GET
+    Returns HTTP 400, 403, 404
```

Comment:
I think the return should be documented as what the json data that would be returned on success.